Makefile

```diff
@@ -4,27 +4,29 @@ WEECHAT_HOME ?= $(HOME)/.weechat
 PREFIX ?= $(WEECHAT_HOME)
 PYTHON ?= python
 
-lib := $(patsubst matrix/%.py, $(DESTDIR)$(PREFIX)/python/matrix/%.py, \
+INSTALLDIR := $(DESTDIR)$(PREFIX)/python/matrix
+
+lib := $(patsubst matrix/%.py, $(INSTALLDIR)/%.py, \
 	 $(wildcard matrix/*.py))
 
 .PHONY: help
 help:
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-install: install-dir install-lib ## Install the plugin to $(DESTDIR)/$(PREFIX)
+install: install-lib | $(INSTALLDIR) ## Install the plugin to $(DESTDIR)/$(PREFIX)
 	install -m644 main.py $(DESTDIR)$(PREFIX)/python/matrix.py
```

Comment:
Wouldn't we want to use the `$(INSTALLDIR)` here as well?

Reply:
`INSTALLDIR := $(DESTDIR)$(PREFIX)/python/matrix`

I opted not to use it here since `matrix.py` isn't actually in the `matrix` directory. If I were to use it here it would look like:
```
install -m644 main.py $(INSTALLDIR).py
```
Which would do the same thing, but is a bit confusing since we are just adding an extension to what should be a directory path

Reply:
Right, I guess this is fine then.